### ISSUE TITLE
Show number of unread comments

### ISSUE THREAD

**Issue by faboweb** [cosmos/cosmos-forum#47]
Peng suggested following format:
14 comments (3 new)

One concern:
Calculating the unread comments (per User) means to process all comments for all threads visible everytime they are shown. I would defer this for now. Alternatively we could sort the threads by "updatedAt".

What do you think @nylira

**Comment by nylira** [comment]
Yeah that is a lot of overhead. Showing unread comments is a quality of life change that we don't need yet. Let's keep this on the backburner for now.
